fuzz_lightyear/datastore.py

```diff
@@ -194,6 +195,7 @@ def wrapped(*args: Any, **kwargs: Any) -> Any:
 
             kwargs[arg_name] = value
 
+        func._arg_cache = [args, kwargs]  # type: ignore
```

Comment:
Does making `_fuzz_cache` a Dict from `(args, kwargs) -> whatever` work here? It would remove 1 variable and allow us to cache multiple things.